pkg/pillar/pubsubtypes/verifer_types.go

```diff
@@ -0,0 +1,43 @@
+package pubsubtypes
+
+import (
+	"github.com/lf-edge/eve/pkg/pillar/pubsub"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"log"
+)
+
+type CallbackFn func(types.AppInstanceConfig) bool
+
+type AppInstanceConfigPubSubBase struct {
+	agentName string
+	db        map[string]types.AppInstanceConfig
+}
+
+func (config *AppInstanceConfigPubSubBase) Get(key string) types.AppInstanceConfig {
+	return config.db[key]
```

Comment:
That can panic. You want something like:
if res, ok := config.db[key]; ok { return res } else { return nil }